packages/react/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {
+  Root,
+  List,
+  Trigger as RadixTab,
+  Content as Panel,
+} from '@radix-ui/react-tabs';
+import { TabsProps, TabItemProps } from '../types';
+import { Flex } from '../Flex';
+import { ComponentClassNames } from '../shared/constants';
+import classNames from 'classnames';
+import { convertStylePropsToStyleObj, prefixer } from '../shared/utils';
+
+const isTabsType = (child: any): child is React.Component<TabItemProps> => {
+  return (
+    typeof child === 'object' &&
+    child.hasOwnProperty('props') &&
+    child.props.title != null &&
+    child.props.children != null
+  );
+};
+export interface Tabs {
```

Comment:
Although TypeScript doesn't complain, I don't think the interface and the component should both be named `Tabs`.

Reply:
In most cases I would agree, but for function interfaces, I think it's logical for them to have the same name